src/lager_nt_eventlog_backend.erl

```diff
@@ -4,19 +4,30 @@
 
 -behaviour(gen_event).
 
--export([init/1, handle_call/2, handle_event/2, handle_info/2, terminate/2,
-        code_change/3]).
+-export([init/1, handle_call/2, handle_event/2, handle_info/2, terminate/2, code_change/3]).
+-export([config_to_id/1]).
 
--record(state, {handle, level}).
+-record(state, {id, handle, level, formatter, format_config}).
 
+-define(DEFAULT_FORMAT, [{pid, ""}, {module, [{pid, ["@"], ""}, module, {function, [":", function], ""}, {line, [":",line], ""}], ""}, " ", message]).
 -include_lib("lager/include/lager.hrl").
 
 %% @private
 init([Source, Level]) ->
-    case nt_eventlog:register_event_source(Source) of
-        {ok, Handle} ->
-            {ok, #state{level=lager_util:level_to_num(Level),
-                        handle = Handle}};
+    init([Source, Level, {lager_default_formatter, ?DEFAULT_FORMAT}]);
+init([Source, Level, {Formatter, FormatterConfig}]) ->
```

Comment:
I would note this in the README; both the default format as well as the option to include a custom one.